__tests__/app.test.js

```diff
@@ -3,20 +3,21 @@ const setup = require('../data/setup');
 const request = require('supertest');
 const app = require('../lib/app');
 const User = require('../lib/models/user');
+const { insert } = require('../lib/models/user');
 
 describe('user account blog-site routes', () => {
   beforeEach(() => {
     return setup(pool);
   });
 
   it('creates a user account on blog site via POST', async () => {
-    const user = {
+    const user = await insert({
```

Comment:
The reason this test is failing (it's returning `{ id: '2' }` instead of 1) is that you're creating a user account here on line 14 with the insert method, but then when you hit the `post('/') ` endpoint on line 22 you're creating *another* user account as part of that action. `res` is set to the result of that post request, so it's always going to come back as id 2 here. I'd recommend omitting the `insert` method of user creation and passing all the same info into the `.send` part of the post request.